### ISSUE TITLE
The block that returns the current name of the component

### ISSUE THREAD

**Issue by AlexITMan** [thunkable/thunkable-issues#194]
Component references are UUIDs, but to get a reference to an object, it is more convenient to use the name of this object.

Suppose there are 5 buttons on the screen with the names btn1..btn5. How to change btn3 button text? This task is easily solved if the object link can be obtained by the name of the button. But when using a UUID, you will have to create additional code each time. I would do this code like this:

when the application started, in the Color property for all buttons I placed their name, and then in a loop on all buttons I looked for a button with the btn3 value in the Color property and when I found it I returned this button UUID.

**Comment by mark-friedman** [comment]
We are working on adding a block to obtain the name of a component.

**Comment by nicbyron** [comment]
@mark-friedman excited for this feature!

**Comment by mark-friedman** [comment]
FWIW, you can refer to components by their names in the various blocks of the "Any" category, so for the specific case mentioned by @AlexITMan you can just generate the names btn1...btn5 and use the appropriate generic setter block.

**Comment by nicbyron** [comment]
@mark-friedman yes, this is a very helpful feature. Excited for the block to obtain the name of a component for easier logic with the Any Button Click block, for example.

**Comment by mark-friedman** [comment]
@nicbyron Just curious how you are intending to use such a block?
